src/components/DeeplinkWrapper/index.website.js

```diff
@@ -33,6 +38,14 @@ class DeeplinkWrapper extends PureComponent {
         return !Browser.isMobile() && typeof navigator === 'object' && typeof navigator.userAgent === 'string' && /Mac/i.test(navigator.userAgent) && !/Electron/i.test(navigator.userAgent);
     }
 
+    // A function to detect if current route should be opened in Web only.
+    isUnsupportedDeeplinkRoute() {
+        return _.some([CONST.REGEX.ROUTES.UNLINK_LOGIN], (unsupportRouteRegex) => {
+            const routeRegex = new RegExp(unsupportRouteRegex);
+            return routeRegex.test(window.location.pathname);
+        });
+    }
```

Comment:
The method name looks sufficient. Instead, we could have a comment inside explaining why the unlink login path was added here. Thoughts?

cc: @hoangzinh @francoisl

Reply:
(Personal preference) I don't mind the comment outside the function, even if the name sounds sufficient. But overall neutral, I'm ok if we keep it or remove it.

However, yes a comment inside explaining _why_ the unlink route is there sounds like a good idea to me!